### PR TITLE
Added replayer loop logic.

### PR DESCRIPTION
**Pull request by alison-li**

### CODE REVIEW

File: provdebug/pvdebug.py [hunk 1]
```diff
@@ -65,19 +75,49 @@ def run():
...
     #args = parser.parse_args(rest)
     
     args = parser.parse_args()
     provdb_command = sys.argv[1]
-    print(args.file)
     if provdb_command == "-r":
-        prov.Replayer(args.file)
-        # TODO: execute path in ProvReplay
+        replayer = prov.Replayer(args.file)
+        print("🥺 Welcome 🥺")
```
Comment (alison-li) [inline provdebug/pvdebug.py:90]: 🥺

**Comment by alison-li** [comment]
@jyoo980 Don't merge yet! I think the placement while loop we added for controlling the flow for `provdb -r <debugging trace file>` actually seems to be preventing `provdb -f <prov JSON file>` from working. Let me know if you're getting the same problem.

Steps to reproduce:
1. Checkout this branch.
2. Try running `provdb -f prov.json`. The Multilingual Debugger doesn't start up.
3. Try running `provdb -r debugging-trace-#.replay`. The Multilingual Debugger in replayer mode does start up.
-------
EDIT: Ignore me, I am dumb. I needed to indent the `return` statement at the exit of the loop.

**Comment by jyoo980** [comment]
@Alison-Li 

You're not dumb, semantically-significant whitespace becomes a problem as files get larger. Thanks for looking into this!

**Review by alison-li — COMMENTED** [review]
Great job @ me

**Review by jyoo980 — APPROVED** [review]
Will change the intro text later on, but let's keep the 🥺 for now

**Pull request merged**